code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Route, Link, useParams } from 'react-router-dom';
+
+
+import './moviedetails.css';
+import BackArrow  from '../components/BackArrow';
+
+
+export const MovieDetails = () => {
+  
+  const { movieId } = useParams();
+
+  const API_KEY = "ab5cef50889fc06c1a1e75470eefe28b";
+  const URL_DETAILS =`https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`;
+  
+  const [movieDetails, setMovieDetails] = useState({});
+    useEffect(() => {
+      fetch(URL_DETAILS)
+      .then(res => res.json())
+      .then(data => {
+        setMovieDetails(data)
+    });
+  }, [URL_DETAILS, movieId]);
+  // }, [URL_DETAILS,movieId]) it asked specififically for the url dependendencie as movieId array?
+
+  // Ensure we have details to paint. If not, do not paint anything. I could have loading hereIn order to avoid "404 error undefined" in console during fetch.
+  if (!movieDetails.backdrop_path) {
+    return null;
+  }
+
+    return (
+      <article className="details_page">
+        <Route path='/movies'>
```

Comment:
Hmm, you don't need `<Route />` over here, let's remove it

Reply:
I can't remember what I was trying to do, may have confused it with Link. 
Removed also in import.